docs/02-for-app-authors/03-metainfo-guidelines/01-quality-guidelines.md

```diff
@@ -30,6 +30,8 @@ Use Flathub verification to indicate that you are the author of the app and that
 
 Apps should set a [brand color](/docs/for-app-authors/metainfo-guidelines/#brand-color) (in two variants, light and dark). This color is used by Flathub and native app store clients on banners, app pages, etc.
 
+Make sure the colors are not too similar to the app icon in order to ensure contrast when the color is used as a background behind the icon. Avoid using light colors color for the dark variant, or the same color for both dark and light. In most cases the dark variant should be a darker more muted version of the light brand color.
+
```

Comment:
An example image would be nice I think - here's a mockup we did to choose the best background colors. Something along those lines could work.

![image](https://github.com/flathub-infra/documentation/assets/5943908/e5d8df47-1c57-4cc0-9151-5244c266d159)